pkg/kubelet/kubelet.go

```diff
@@ -1943,3 +1947,222 @@ func (kl *Kubelet) GetRootInfo(req *cadvisorApi.ContainerInfoRequest) (*cadvisor
 func (kl *Kubelet) GetMachineInfo() (*cadvisorApi.MachineInfo, error) {
 	return kl.cadvisor.MachineInfo()
 }
+
+func (kl *Kubelet) SyncRocketPods(allPods []api.BoundPod, podSyncTypes map[types.UID]metrics.SyncPodType, start time.Time) error {
```

Comment:
Why is there rocket specific implementation in the kubelet?
I would expect all this to be in the rocket implementation of `container.Runtime`.